src/test/java/com/webank/webase/front/util/SampleKeys.java

```diff
@@ -0,0 +1,30 @@
+package com.webank.webase.front.util;
+
+import java.math.BigInteger;
+
+import org.fisco.bcos.web3j.crypto.Credentials;
+import org.fisco.bcos.web3j.crypto.ECKeyPair;
+import org.fisco.bcos.web3j.utils.Numeric;
+
+/** Keys generated for unit testing purposes. */
+public class SampleKeys {
+
+    public static final String PRIVATE_KEY_STRING =
+            "a392604efc2fad9c0b3da43b5f698a2e3f270f170d859912be0d54742275c5f6";
+    static final String PUBLIC_KEY_STRING =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)